docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -168,8 +171,34 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 <1> `HotBackup` resource name used for restore. The backup folder name will be taken from the `HotBackup` resource.
 <2> Agent responsible for restoring data from the local storage.
-    The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
-    uses the latest agent version that is compatible with its version.
+The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
+uses the latest agent version that is compatible with its version.
+
+To restore from local backups that were not created by the HotBackup resource, configure `localConfig`.
```

Comment:
We should have titles or at least listing approach to emphasize the two different restore scenario when we have local backup. I don't know if it makes sense for you but I suggest something like this.
- Local Backups taken with the Operator.
- Local Backups taken with the Helm Chart 

While thinking about their difference, I bring another idea. How about creating new page just for the newly added restore support? Since backup and restore features are expected as one of the main features which will be used always by the all customers who are using operator. However, newly supported restore is not actually a feature which will be used continuously. Instead, it's kinda migration solution for older operator versions and helm chart users. Also in the future we may decide to support fully custom backups. Are we going to put it also in this page? I don't think so, it'll be a migration solution as well so I suggest to separate new restore option from the existing one. Sorry for late feedback, I just got into this idea.

Reply:
Such a nice a feedback Cagri, let's talk with the team and decide together. I am not sure %100 but it might be a good idea to make it a part of migration documents.

Reply:
A better idea would be to put the documentation about localConfig in here, and reference it from migration documentations.

Reply:
I committed more changes to improve the docs: https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/171/commits/32bb2ddc3815561064cb9d4cdf54d7fd49800fe4. I didn't mention about the helm and migration words explicitly, because in the feature we will have specific guides for those scenarios like https://github.com/hazelcast/hz-docs/pull/1052. We will refer to this document from them.